index.js

```diff
@@ -29,12 +29,17 @@ const startGame = numOfPlayers => {
     let playersPoints = new Array(numOfPlayers);
     let unveiled = 10;
     let timeouted = false;
+    //TODO(Moty) this below should be a template.
```

Comment:
i'm not familiar with templates. just to make sure i read what you mean (and change respectfully) - can you leave a link?